### ISSUE TITLE
Proper Support for Dark Mode

### ISSUE THREAD

**Issue by FlorianRappl** [smapiot/piral-inspector#1]
Right now our styling is not sensitive to dark mode. We should detect this and switch colors.

```css
/* Dark mode */
@media (prefers-color-scheme: dark) {
    body {
        background-color: #000;
        color: white;
    }
}
```

Also, by default, we should set the background to white - to avoid any issues with other themes.
